site/gatsby-site/migrations/2023.09.14T18.44.03.create-default-user.js

```diff
@@ -0,0 +1,79 @@
+const { ObjectId } = require('bson');
+
+const config = require('../config');
+
+const userId = '65031f49ec066d7c64380f5c';
+
+/**
+ *
+ * @param {{context: {client: import('mongodb').MongoClient}}} context
+ */
+
+exports.up = async ({ context: { client } }) => {
+  const first_name = 'Anonymous';
+
+  const last_name = 'User';
+
+  await client.connect();
+
+  const usersCollection = client.db(config.realm.production_db.db_custom_data).collection('users');
+
+  const defaultUser = await usersCollection.findOne({ userId: userId });
+
+  if (defaultUser) {
```

Comment:
Is this because it already exists in staging and not production? It is a bit weird to have a condition like this in a migration 🤔

Reply:
Actually the opposite, this user already exists on Prod (because Kevin created it manually to get the user ID, which is correct), but not in Staging or any local env.

Reply:
What about restoring staging to prodction before merging this one so we are sure it works?

Otherwise will be testing it live.

Reply:
I took that approach into consideration, but the Github Action for restoring the database doesn't include the users' collection (`customData` db). We should restore the users' collection manually.
I'm OK doing it that way.

Reply:
I prefer to do that manually and get the peace of mind of testing this first in staging

Reply:
+1

Reply:
Done! The `users` collection was restored from Production to Staging.